pkg/loop/logger.go

```diff
@@ -52,6 +52,14 @@ func removeArg(args []interface{}, key string) ([]interface{}, string) {
 	return args, ""
 }
 
+func logDebug(msg string, l logger.Logger, args ...interface{}) {
```

Comment:
let's add a comment about what we are trying to intercept panics here. IIRC you said something like if hashicorp can't parse a msg, then it falls back to logging that message at Debug. we need to capture that context in a comment here

Reply:
Addressed in [`bc1e8fa` (#384)](https://github.com/smartcontractkit/chainlink-common/pull/384/commits/bc1e8fa5dc95420d53d217baca37818f5b266eef)